Tella/Data/Database/ReportDatabase.swift

```diff
@@ -0,0 +1,393 @@
+////
+////  UwaziReportDatabase.swift
+////  Tella
+////
+////  Created by Robert Shrestha on 9/14/23.
+////  Copyright © 2023 HORIZONTAL. All rights reserved.
+////
+//
+//import Foundation
+//
+//extension TellaDataBase {
+//    func createReportTable() {
```

Comment:
I think there are some classes are still in the project. Could you remove the file